Source/Request.swift

```diff
@@ -346,18 +340,30 @@ open class Request {
         retryOrFinish(error: self.error)
     }
 
-    /// Called when the `RequestDelegate` is retrying this `Request`.
+    /// Called when the `RequestDelegate` is retrying this `Request`. Calls `reset()`.
     func requestIsRetrying() {
```

Comment:
We may want to rename this to better reflect what it's used for. Perhaps `prepareForRetry`?

Reply:
Good call, will update.

Reply:
Renamed in fea5fc86.